src/components/CopyToClipBoard/CopyToClipBoard.test.tsx

```diff
@@ -1,28 +1,27 @@
-import { ReactWrapper } from 'enzyme';
 import React from 'react';
 
 import { copyToClipBoardUtility } from 'verdaccio-ui/utils/cli-utils';
-import { mount } from 'verdaccio-ui/utils/test-enzyme';
+import { render, cleanup, fireEvent } from 'verdaccio-ui/utils/test-react-testing-library';
 
 import CopyToClipBoard from './CopyToClipBoard';
-import { CopyIcon } from './styles';
 
 jest.mock('verdaccio-ui/utils/cli-utils');
 
 describe('<CopyToClipBoard /> component', () => {
-  let wrapper: ReactWrapper;
+  let wrapper: any;
```

Comment:
Why this type is any now?

Reply:
Not sure yet, I just wanna remove enzyme. We will figure out.